python/pyarrow/tests/test_dataset_encryption.py

```diff
@@ -15,10 +15,14 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import base64
 from datetime import timedelta
+import numpy as np
 import pyarrow.fs as fs
 import pyarrow as pa
+import pyarrow.parquet as pq
```

Comment:
I'm not sure this will work on builds where Parquet is disabled. Let's watch for CI.

Reply:
We might not have an actual test build without parquet .. (but indeed in general we always import pyarrow.parquet within a try/except to let this run on versions without parquet support as well)

Reply:
Ah, I thought we did...

Reply:
I was just deducting that from the fact we got no failures in the builds you triggered. 
But looking at our tasks.yml, it seems the python minimal builds are not included in the "python" group ...

Reply:
That would be why :-(

Reply:
Testing and potentially fixing in https://github.com/apache/arrow/pull/40505 (and at once adding the minimal tests to the python group)